tests/test_sorting.py

```diff
@@ -147,3 +207,8 @@ def test_error_empty(client):
         400,
         [{"code": "invalid_sort", "source": {"parameter": "sort"}}],
     )
+
+
+def test_duplicate_fields(client):
+    with pytest.raises(AssertionError):
```

Comment:
nitpick: Together with my other suggestion:

```suggestion
    with pytest.raises(ValueError, match="Sort field(s) cannot be passed as both positional and keyword arguments"):
```